modules/base/dashboard/dashboarditemdistance.cpp

```diff
@@ -358,6 +359,8 @@ DashboardItemDistance::DashboardItemDistance(const ghoul::Dictionary& dictionary
     addProperty(_requestedUnit);
 
     _font = global::fontManager.font(_fontName, _fontSize);
+
+    _buffer.resize(256);
```

Comment:
This should probably become a std::array instead of a vector since we are not using the resizing?